### PR TITLE
OSDOCS-12677: Edit ROSA Autoscaling Docs

### PR DESCRIPTION
**Pull request by jneczypor**
[OSDOCS-12677](https://issues.redhat.com//browse/OSDOCS-12677): Edit ROSA Autoscaling Docs

Version(s):
4.17+

Issue:
https://issues.redhat.com/browse/OSDOCS-12677

Link to docs preview:

- https://85519--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_cluster_admin/rosa-cluster-autoscaling.html#rosa-enable-cluster-autoscale-cli-after_rosa-cluster-autoscaling 

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by kmcolli** [comment]
Looks good to me

**Review by mletalie — COMMENTED** [review]
/label peer-review-in-progress

File: modules/rosa-cluster-autoscaler-cli-after.adoc [hunk 1]
```diff
@@ -6,6 +6,11 @@
 [id="rosa-enable-cluster-autoscale-cli-after_{context}"]
 = Enable autoscaling after cluster creation with the ROSA CLI
 
+[IMPORTANT]
+====
+The following procedure is only supported for ROSA Classic clusters. To enable autoscaling after cluster creation for ROSA with HCP clusters, see link:https://docs.openshift.com/rosa/rosa_cluster_admin/rosa_nodes/rosa-nodes-about-autoscaling-nodes.html[Enabling autoscaling nodes on a cluster].
```
Comment (mletalie) [inline modules/rosa-cluster-autoscaler-cli-after.adoc:11]: Looks good! Only thing I would suggest is to change the link to _https://docs.openshift.com/rosa/rosa_cluster_admin/rosa_nodes/rosa-nodes-about-autoscaling-nodes.html#nodes-enabling-autoscaling-nodes_ or change the link text to "About autoscaling nodes on a cluster"...so the target matches the link....

File: modules/rosa-cluster-autoscaler-cli-after.adoc [hunk 2]
```diff
@@ -25,3 +30,6 @@ $ rosa create autoscaler --cluster=<mycluster>
 ----
 $ rosa create autoscaler --cluster=<mycluster> <parameter>
 ----
+
+.Next steps
+* link:https://docs.openshift.com/rosa/rosa_cluster_admin/rosa-cluster-autoscaling.html#rosa-edit-cluster-autoscale-cli_rosa-cluster-autoscaling[Enabling autoscaling after cluster creation with the ROSA CLI]
```
Comment (mletalie) [inline modules/rosa-cluster-autoscaler-cli-after.adoc:35]: Maybe change "Enabling" to "Edit" for target to match link?

**Review by mletalie — COMMENTED** [review]
/lgtm
/remove-label peer-review-needed
/remove-label peer-review-in-progress
/label peer-review-done

**Comment by mletalie** [comment]
Just a couple of suggestions, otherwise LGTM!

**Review by EricPonvelle — COMMENTED** [review]
/lgtm

**Pull request merged**